openexr-sys/c_wrapper/cexr.cpp

```diff
@@ -127,7 +127,6 @@ int CEXR_Header_get_channel(const CEXR_Header *header, const char name[], const
         *out = channel_ptr;
         return 0;
     } else {
-        *err_out = "no channel found with that name";
         return 1;
```

Comment:
Is there a good reason not to just return a possibly-null pointer directly here?